tests/tx/test_tx_storage.py

```diff
@@ -146,11 +146,11 @@ def test_storage_basic(self):
             self.assertEqual(2, self.tx_storage.get_tx_count())
             self.assertEqual(3, self.tx_storage.get_count_tx_blocks())
 
-            block_parents_hash = [x.data for x in self.tx_storage.get_block_tips()]
+            block_parents_hash = self.tx_storage.get_best_block_tips()
```

Comment:
I feel we should check all block tips here (not only the ones in the best chain).

Reply:
I'm splitting this into two tests: the same as it was before and one with the new indexes. I don't think we have "all block tips" in the new indexes, only the candidates to be block parents for the next block.

Reply:
Good one. Thanks!